tests/Foundation/FoundationAliasLoaderTest.php

```diff
@@ -20,4 +20,10 @@ public function testGetInstanceCreatesOneInstance()
         $loader = AliasLoader::getInstance(['foo' => 'bar']);
         $this->assertSame($loader, AliasLoader::getInstance());
     }
+
+    // unregister dummy not existing classes
```

Comment:
these part of the code is simple bugfix since phpunit bloat the class alias memory of php with fake class which are later not usable. and without unregister they can unloaded.